database/migrations/2021_05_25_134101_create_gadgets_table.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateGadgetsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('gadgets', function (Blueprint $table) {
```

Comment:
Willen we wel `gadgets` dynamisch maken? Want de gadgets worden dus ook geseed, maar verder zie ik ook de naam van een gadget als een statische waarde terugkomen in het project. Daar ging mijn eerder opmerking over dat het net zo goed een `enum` kan zijn. Wat heeft het voor meer waarde als dit een tabel is met naam?
Ik zou het bij de tabel `gadgets_users` houden en daar dan de naam van een gadget plaatsen.

Reply:
We hadden het eigenlijk als lookup table gemaakt. Er zijn natuurlijk andere structuren mogelijk, maar dit is de manier waarp wij het hadden aangepakt. Ik zie persoonlijk ook geen echte performance gains door het aan te passen. Eventueel kan het later zo gemaakt worden dat de statische waarden automatisch worden gezet naar de waarden in de lookup table, maar verder denk ik dat het op deze manier wel werkt.